### PR TITLE
Developer docs improvements

### PR DESCRIPTION
**Pull request by alenavolk**
I just finished building a Bodhi development environment on a clean Fedora 26. Here are some tweaks I made to get it up and running. I hope they will save other newcomers some time.

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

File: requirements.txt [hunk 1]
```diff
@@ -17,6 +18,7 @@ progressbar
 pyDNS
 pylibravatar
 pyramid
+pyramid_debugtoolbar
```
Comment (bowlofeggs) [inline requirements.txt]: This is not a production requirement, and in fact you wouldn't want it present on production. If we add this to this file, then the production package won't install without this dependency so we should drop this line.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
Thanks for the patches! Just one issue to fix and then we can merge.

File: requirements.txt (hunk 1 above)
Comment (alenavolk) [reply requirements.txt]: Thanks for the feedback! I moved it to developer docs.

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
This patch is included in the bodhi-3.1.0-0.1.beta, which is deployed to [staging](https://bodhi.stg.fedoraproject.org/) and is also available via [Copr](https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release/).

**Pull request merged**